projects/packages/forms/src/dashboard/hooks/use-inbox-data.ts

```diff
@@ -47,6 +47,22 @@ interface UseInboxDataReturn {
 	filterOptions: Record< string, unknown >;
 }
 
+const RESPONSE_FIELDS = [
+	'id',
+	'status',
+	'date',
+	'date_gmt',
+	'author_name',
+	'author_email',
+	'author_url',
+	'author_avatar',
+	'ip',
+	'entry_title',
+	'entry_permalink',
+	'has_file',
+	'fields',
```

Comment:
This will need the new is_unread field. depending on which one lands first. See https://github.com/Automattic/jetpack/pull/45350/files#diff-9db38de6e3678fb163907d4d47e3b1f38a9a74e038e9b5b9f475c190c0e8a937R514